.github/workflows/ci.yml

```diff
@@ -121,3 +118,20 @@ jobs:
       - name: Build files
         run: npm run build
         working-directory: search/search-dev-tools
+
+      - name: Check the diff
+        run: |
+          if [ "$(git diff --ignore-space-at-eol search/search-dev-tools/ | wc -l)" -gt "0" ]; then
+            echo "::warning::There are changes in the built files."
+            if [ "${{ github.event.head_commit.committer.email }}" != "github-actions@github.com" ]; then
```

Comment:
This is to avoid infinite loops if something goes terribly wrong. As an alternative, we can check that `${{ github.actor }}` is not the name of the bot. However, there could be conflicts if we add some other automation later, as the name of the bot cannot be changed but the email address of the committer can.